static/specification.html

```diff
@@ -144,289 +144,296 @@ <h2>
     </section>
     <section>
       <h2>
-        Extensions to the `Document` interface
+        Extensions to the `Navigator` interface
       </h2>
       <pre class="idl">
-        partial interface Document {
+        partial interface Navigator {
           readonly attribute Monetization monetization;
         };
       </pre>
     </section>
-    <section id="flow">
+    <section>
       <h2>
-        Flow
+        Link type "monetization"
       </h2>
+      <aside class="issue">
+        This is a "monkey patch" for the HTML spec, which means this would
+        eventually be moved to the HTML spec.
+      </aside>
+      <p>
+        The monetization keyword indicates a [=payment end-point=] used to
+        monetize the document.
+      </p>
+      <aside class="issue" data-number="192"></aside>
+      <p>
+        Every instance of {{Navigator}} has an <dfn data-dfn-for=
+        "Navigator">[[\activeMonetizationElement]]</dfn> internal slot of type
+        {{HTMLLinkElement}}, which represents the element in the document which
+        controls the monetization. It is initialized as `null`.
+      </p>
+      <p>
+        The <code data-x="rel-monetization">monetization</code> keyword may be
```

Comment:
What is this `data-x` thing?

Reply:
This is watsi. This will eventually move to the HTML spec.